homeassistant/components/igloohome/strings.json

```diff
@@ -0,0 +1,26 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Add API Access Credentials",
+        "description": "Set your account's API keys to enable Home Assistant to poll the status of your devices.",
+        "data": {
+          "client_id": "Client ID",
+          "client_secret": "Client Secret"
+        },
+        "data_description": {
+          "client_id": "Client ID provided by your iglooaccess account.",
+          "client_secret": "Client Secret provided by your iglooaccess account."
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "Client ID is already in use."
```

Comment:
I'd go with the default reference, which shows "this account is already configured", which is IMO more clear about what happened than "is already in use". e.g. Did someone else use it?